scripts/findChrome.js

```diff
@@ -197,6 +198,24 @@ async function downloadChromium() {
   }
 }
 
+function isSuitableVersion(executablePath) {
+  let versionOutput
+  // in case installed Chrome is not runnable
+  try {
+    versionOutput = execSync(`${executablePath} --version`).toString()
```

Comment:
Let’s use async `exec` if we already have `async`/`await` in the code (so using async code till be very easy)

Reply:
Done.